Update ActiveSupport dependency to 4.2.x

conroywhitney opened the pull request.

conroywhitney committed:
Update ActiveSupport dependency to 4.2.x

dillonwelch commented:
Looks fine to me.

I'm going to look into hooking this up to CI, so we have at least some validation that things still work here.

Commit:
Dummy commit to re-trigger CircleCI

Commit:
Add `circle.yml` to force ruby version on CircleCI

After upgrading to ActiveSupport 4.2.x, the `tins` gem requires `ruby 2+`. In order to get CircleCI to run green, tell CircleCI to use Ruby 2.2.3 instead of default 1.9.3.
